sdk/metric/benchmark_test.go

```diff
@@ -230,11 +230,11 @@ func BenchmarkFloat64LastValueAdd(b *testing.B) {
 
 // Histograms
 
-func benchmarkInt64HistogramAdd(b *testing.B, name string) {
+func BenchmarkInt64HistogramAdd(b *testing.B) {
 	ctx := context.Background()
 	fix := newFixture(b)
 	labs := makeLabels(1)
-	mea := fix.meterMust().NewInt64Histogram(name)
+	mea := fix.meterMust().NewInt64Histogram("int64.histogram")
```

Comment:
Does the name suffix hardcode something? Not clear how changing the name fixes this.

Reply:
See https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/metric/processor/processortest/test.go#L182-L207